src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE

```diff
@@ -230,30 +230,30 @@ maybe(
 maybe(
     http_archive,
     name = "remote_java_tools_linux",
-    sha256 = "ea2ee2f9eab37e04eae5aa8a266373132fe57676c25ea033504bc5438d6bc1ba",
+    sha256 = "37acb8380b1dd6c31fd27a19bf3da821c9b02ee93c6163fce36f070a806516b5",
     urls = [
-        "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_linux-v5.1.zip",
-        "https://github.com/bazelbuild/java_tools/releases/download/javac11-v5.1/java_tools_javac11_linux-v5.1.zip",
+        "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_linux-v6.0.zip",
+        "https://github.com/bazelbuild/java_tools/releases/download/javac11-v5.1/java_tools_javac11_linux-v6.0.zip",
```

Comment:
First we have to test the RC.

I recommend doing with these two global replacements in the bazel repo:

```
bazel_java_tools/releases -> bazel_java_tools/release_candidates
v6.0 -> v6.0-rc1
v5.1 -> v6.0-rc1
```
You comment out the github urls until we publish the release on github.

Reply:
You'll end up for example with:

[https://mirror.bazel.build/bazel_java_tools/release_candidates/javac11/v6.0/java_tools_javac11_linux-v6.0-rc1.zip](https://mirror.bazel.build/bazel_java_tools/release_candidates/javac11/v6.0/java_tools_javac11_linux-v6.0-rc1.zip) which is a valid URL

Reply:
Sorry, better try these replacements though (in this order):

```
v6.0 -> v6.0-rc1
bazel_java_tools/releases/javac/v5.1 -> bazel_java_tools/release_candidates/javac/v6.0
```